File: Java/Employee.java
import java.util.Date;

public class Employee  {
    Integer id;
    String jobPosition;
    Date startDate;
    Date endDate;
    Person person;
    Company company;

    public Employee(String jobPosition, Person person, Company company){
        this.jobPosition = jobPosition;
        this.person = person;
        this.company = company;
    }

    public void printEmployee(){
        System.out.println("Person Name: " + person.name);
        System.out.println("    Company     : " + company.bussinessName);
        System.out.println("    Job Position: " + this.jobPosition);
    }
}

File: Python/employee.py
from company import Company
from person import Person

class Employee:
    id              = int
    jobPosition     = str
    company         = Company("", 0)
    person          = Person("","",0)
    startDate       = str
    endDate         = str

    def __init__(self, jobPosition, company, person):
        self.jobPosition = jobPosition
        self.company = company
        self.person = person


    def printEmployee(self):
        print(self.jobPosition)
        print(vars(self.company))
        print(vars(self.person))